nautobot/extras/management/commands/runjob.py

```diff
@@ -74,23 +74,26 @@ def handle(self, *args, **options):
             job_result = JobResult.objects.get(pk=job_result.pk)
 
         # Report on success/failure
-        for test_name, attrs in job_result.data.items():
-
-            if test_name in ["total", "output"]:
-                continue
+        groups = set(JobLogEntry.objects.filter(job_result=job_result).values_list("grouping", flat=True))
+        for group in sorted(groups):
+            logs = JobLogEntry.objects.filter(job_result__pk=job_result.pk, grouping=group)
```

Comment:
This smells like possible bad query performance to me. Have you considered the following?

- Ordering the query by `grouping` to eliminate the need to use `sorted(groups)`?
- Using `prefetch_related` on `job_result` to get all of these objects in a single query, instead of round trips to the database for each group?

Reply:
I was asked to add sorting to as a set does not have an order. I could change the filter, but it would still be in an unordered set.  @glennmatthews was concerned about the sorting.

I will look at the `prefetch_related`.

Reply:
Also to note, that this is only for running the jobs from the CLI, which I would think is rare.